src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -273,11 +273,12 @@ trait AttributeComponent {
       }
 
       def dropAttributeTempTableAction() = {
-        sql"""drop table #${baseTableRow.tableName}_TEMP""".as[Int]
+        sql"""drop temporary table if exists #${baseTableRow.tableName}_TEMP""".as[Int]
       }
 
       def upsertAction(ownerIds: Seq[OWNER_ID], insertFunction: () => ReadWriteAction[Unit]) = {
-        createAttributeTempTableAction() andThen
+        dropAttributeTempTableAction() andThen
```

Comment:
1 - I think the error would wind up somewhere in the JDBC connection but I have no idea how to extract it if so. IMO the log shouldn't be necessary if the `drop if exists` handles previous errors gracefully.

The only thing to sanity check here is that no two threads/txns will ever simultaneously use the same connection from the connection pool, but I think that in all likelihood the answer to that is a yes.